### PR TITLE
Mark SERVER_ID as deprecated and use LICENSE_ID instead because this less confusing

### PR DESCRIPTION
**Pull request by xiwenc**
Internal processes refer `SERVER_ID` being the computed value of the `UUID` as input. To avoid confusion we will call this `UUID` value `LICENSE_ID` from now on.

### CODE REVIEW

**Comment by djvdorp** [comment]
@xiwenc can you ping me when this could be reviewed and merged?

**Comment by xiwenc** [comment]
@jtwaleson do you approve?

File: tests/run.sh [hunk 1]
```diff
@@ -22,5 +22,9 @@ virtualenv -p python2 venv
 . venv/bin/activate
 pip install -r requirements.txt
 
+echo "Begin clean up of environment"
```
Comment (jtwaleson) [inline tests/run.sh]: won't this remove the possibility to run multiple tests at the same time?

Comment (xiwenc) [inline tests/run.sh]: this removes possibility to run concurrent builds. But multiple tests are still possible.

**Pull request merged**